source/common/event/libevent.h

```diff
@@ -31,6 +31,11 @@ namespace Libevent {
  */
 class Global {
 public:
+  // True if initialized() has been called.
+  static bool INITIALIZED;
```

Comment:
I think this should be lower case, this isn't a constant?

Reply:
nit: This variable can be private.

Reply:
Oh, is the CAPS only for contants?  I thought it was for all static.
Ignoring the naming conflict, static bool intitialized; is weird and static bool initialized_ isn't correct.